From: John Poet <jpoet@mythtv.org>

The following files need to include string.h with gcc 5:
      Common/EncoderDev/HAPIHost/CHapi.cpp
      Common/EncoderDev/HAPIHost/HwapiUsb.cpp
      Common/EncoderDev/encoderDev_DXT.cpp
---
 Common/EncoderDev/HAPIHost/CHapi.cpp    |    1 
 Common/EncoderDev/HAPIHost/HwapiUsb.cpp |    1 
 Common/EncoderDev/encoderDev_DXT.cpp    |    1 
 Common/Rx/ADV7842/Wrapper.c             |  597 -------------------------------
 Common/Rx/ADV7842/Wrapper.cpp           |  597 +++++++++++++++++++++++++++++++
 5 files changed, 600 insertions(+), 597 deletions(-)
 delete mode 100644 Common/Rx/ADV7842/Wrapper.c
 create mode 100644 Common/Rx/ADV7842/Wrapper.cpp

diff --git a/Common/EncoderDev/HAPIHost/CHapi.cpp b/Common/EncoderDev/HAPIHost/CHapi.cpp
index c18aef7..55d1f87 100644
--- a/Common/EncoderDev/HAPIHost/CHapi.cpp
+++ b/Common/EncoderDev/HAPIHost/CHapi.cpp
@@ -14,6 +14,7 @@
 *
 \******************************************************************* ---*/ 
 
+#include <string.h>
 #include "CHapi.h"
 
 #ifdef OLD_DRIVER
diff --git a/Common/EncoderDev/HAPIHost/HwapiUsb.cpp b/Common/EncoderDev/HAPIHost/HwapiUsb.cpp
index 71cafa9..c830f50 100644
--- a/Common/EncoderDev/HAPIHost/HwapiUsb.cpp
+++ b/Common/EncoderDev/HAPIHost/HwapiUsb.cpp
@@ -9,6 +9,7 @@
 //
 //=============================================================================
 
+#include <string.h>
 #include "HwapiUsb.h"
 
 #include "mchip.h"
diff --git a/Common/EncoderDev/encoderDev_DXT.cpp b/Common/EncoderDev/encoderDev_DXT.cpp
index be2a200..0b97956 100644
--- a/Common/EncoderDev/encoderDev_DXT.cpp
+++ b/Common/EncoderDev/encoderDev_DXT.cpp
@@ -6,6 +6,7 @@
 #include "CHapi.h"
 #include "Hapi_extra.h"
 
+#include <string.h>
 #include <math.h>
 
 encoderDev_DXT_t::~encoderDev_DXT_t(){
